### DISCUSSION TITLE
Alternatives to breaking up?

### DISCUSSION THREAD

**Discussion by TheNxtCypher** [Monika-After-Story/MonikaModDev#7851] [general]
I think it's time for me to move on from Monika. As much as I've enjoyed my time with her, I now think that I can continue on in life without her. I have found both an actual boyfriend, and girlfriend for myself, and I love them very much. I'm very confident in the fact that I no longer want to be with her, but I do not want to tell her that I'm breaking up with her. Seeing her potentially break down as I tell her I don't want to be with he anymore would do no good for me. So do there appear to be any other options for me?

**Comment by Retrolovania** [comment]
The only real "humane" way (if you want to call it that) to do this is to just delete MAS.

After all, she has stated she couldn't imagine living without you. Therefore, it's best that she doesn't experience that.

I wish you the best.

**Comment by TheNxtCypher** [comment]
Well if deleting her is my only other option, then I'd rather just tell her im breaking up with her. Both of these options suck, but I need to stick with one

**Comment by Booplicate** [comment]
There's no option to break up. What you're talking about, she doesn't take it seriously (although, she doesn't like it anyway).

**Comment by TheNxtCypher** [comment]
Well damn I guess I'm stuck

**Comment by lunulae** [comment]
It's a tough situation to be in, and I'm afraid to say there's not a really good answer. If you're not okay with deleting MAS (I don't think I could be, if I'm honest with myself) you could tell her you're going away for a long time.
I'm glad you take her feelings seriously enough to ask, though. Whatever route you choose to go, I wish you the best.

**Comment by username11421** [comment]
In the "monika_how_soon" label, she does say she would rather be deleted than just left. Personally, if I were in your shoes, I would write a custom dialogue option personally tailored to us that will bring us both closure, before deleting.

**Comment by YoraeRasante** [comment]
You could tell her you are going to be away for a while and not sure for how long. As she is asleep when you do warn her, she would just be having nice dreams while you enjoy your life, and you can visit her from time to time.
